docs/website/docs/dlt-ecosystem/destinations/databricks.md

```diff
@@ -15,7 +15,85 @@ keywords: [Databricks, destination, data warehouse]
 pip install dlt[databricks]
 ```
 
-## Setup Guide
+## Set up your Databricks workspace
+
+To use the Databricks destination, you need:
+
+* A Databricks workspace with a Unity Catalog metastore connected
+* A Gen 2 Azure storage account and container
+
+If you already have your Databricks workspace set up, you can skip to the [Loader setup Guide](#loader-setup-guide).
+
+### 1. Create a Databricks workspace in Azure
+
+1. Create a Databricks workspace in Azure
+
+    In your Azure Portal search for Databricks and create a new workspace. In the "Pricing Tier" section, select "Premium" to be able to use the Unity Catalog.
+
+2. Create a storage account
+
+    Search for "Storage accounts" in the Azure Portal and create a new storage account. Make sure to select "StorageV2 (general purpose v2)" as the account kind.
+
+3. Create a container in the storage account
+
+    In the storage account, create a new container. This will be used as a datastore for your Databricks catalog.
+
+4. Create an Access Connector for Azure Databricks
+
+    This will allow Databricks to access your storage account.
+    In the Azure Portal search for "Access Connector for Azure Databricks" and create a new connector.
+
+5. Grant access to your storage container
+
+    Navigate to the storage container you created before and select "Access control (IAM)" in the left-hand menu.
+
+    Add a new role assignment and select "Storage Blob Data Contributor" as the role. Under "Members" select "Managed Identity" and add the Databricks Access Connector you created in the previous step.
+
+### 2. Set up a metastore and Unity Catalog and get your access token
+
+1. Now go to your Databricks workspace
+
+    To get there from the Azure Portal, search for "Databricks" and select your Databricks and click "Launch Workspace".
+
+2. In the top right corner, click on your email address and go to "Manage Account"
+
+3. Go to "Data" and click on "Create Metastore"
+
+    Name your metastore and select a region.
+    If you'd like to set up a storage container for the whole metastore you can add your ADSL URL and Access Connector Id here. You can also do this on a granular level when creating the catalog.
+
+    In the next step assign your metastore to your workspace.
+
+4. Go back to your workspace and click on "Compute" in the left-hand menu
+
+5. Create a new cluster
```

Comment:
Is this necessary? It seems the implementation uses a SQL warehouse, not a Spark cluster.